openpos-client-libs/projects/openpos-client-core-lib/src/lib/shared/screen-parts/options-list/options-list.component.ts

```diff
@@ -39,7 +40,7 @@ export class OptionsListComponent extends ScreenPartComponent<OptionsListInterfa
 
     isMobile: Observable<boolean>;
 
-    isFirstElementFocused = false;
+    autoFocusFirstOption;
```

Comment:
can you not just combine this with what's in the ctor?

```ts
autoFocusFirstOption = CONFIGURATION. autoFocusFirstOptionsListOption;
```
If not can we explicitly type this so it's not as confusing?

```suggestion
    autoFocusFirstOption: boolean;
```